panel/models/tabulator.ts

```diff
@@ -903,7 +906,9 @@ export class DataTabulatorView extends HTMLBoxView {
         }
       }
     }
+    this._updating_expanded = true
     this.model.expanded = expanded
```

Comment:
Maybe wrap this in a `try/finally`?

Reply:
Will probably finally write a decorator of some kind for this.